uploadNeuroDB/NeuroDB/FileDecompress.pm

```diff
@@ -8,9 +36,19 @@ use Path::Class;
 use Archive::Extract;
 use Archive::Zip;
 
-################################################################
-#####################Constructor ###############################
-################################################################
+
+=pod
+
+=head3 new($file_path) (constructor)
+
+Create a new instance of this class.
+
+INPUT: path of the file to extract.
+
+RETURNS: a C<Archive::Extract> object on success, or false on failure
```

Comment:
consistency in capitalization of `false` throughout the RETURNS in the overall file.